backend/RoomPlanner/room-planner/src/test/java/edu/roomplanner/RoomPlannerApplicationTests.java

```diff
@@ -12,5 +14,15 @@
 	@Test
 	public void contextLoads() {
 	}
+	@Test
+	public void TestEncryption()
```

Comment:
I have several points for this method:
-  This test should be in a class named EncriptionServiceUnitTest.
-  The method name should suggest the tested behavior.
-  You can extract the tested service into a class attribute.
-  You can name the variable sut (standing for system under test which is a method of highlighting the tested service).